frontend/src/components/react-table/styles/styles.scss

```diff
@@ -0,0 +1,43 @@
+.table {
+  position: absolute;
+  top: 181px;
+  left: 305px;
+  width: 1074px;
+  height: 452px;
+  padding: 15px;
+  background: #242636;
```

Comment:
colors from the design should be declared in one place as a variables and then reused all over the stylesheets